docs/deployment-examples/kubernetes-deployments/kubectl/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: kubectl  
+description: The kubectl utility is required by Octopus Deploy kubernetes integration.  
+position: 100
+---
+
+## kubectl
+
+The [kubectl command-line tool](https://kubernetes.io/docs/reference/kubectl/overview/) is required by Octopus Deploy's kubernetes features.
+
+kubectl is not bundled with Octopus, and must be pre-installed on the [worker](/docs/infrastructure/workers/index.md) or Octopus Server which will execute steps and health checks against a [Kubernetes target](/docs/infrastructure/deployment-targets/kubernetes-target/index.md). Alternatively, [execution containers](/docs/deployment-process/execution-containers-for-workers/index.md) may be used. 
```

Comment:
i think execution containers can work, but they wont work for healthchecks i think

See this thread - https://octopusdeploy.slack.com/archives/C016SSDNTCY/p1597354774145800